nozzle/event_handler.go

```diff
@@ -209,8 +208,8 @@ func (w *eventHandlerImpl) getTags(event *events.Envelope) map[string]string {
 	return tags
 }
 
-func (w *eventHandlerImpl) sendMetric(name string, value float64, ts int64, source string, tags map[string]string) {
-	if debug {
+func (w *EventHandler) sendMetric(name string, value float64, ts int64, source string, tags map[string]string) {
+	if trace {
```

Comment:
Also wrap line 228 around a debug flag. i.e log individual send failures only if debug flag is set. Either ways we will always emit a send failure metric for debugging purposes.